### PR TITLE
Add GraalPy guides

### PR DESCRIPTION
**Pull request by timfel**
- [x] Deal with all "todo" left in the guides.

### CODE REVIEW

File: graalpy/graalpy-javase-guide/README.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+## 2. What you will need
+
+To complete this guide, you will need the following:
+
+ * Some time on your hands
+ * A decent text editor or IDE
+ * A suppported JDK[^1], preferably the latest [GraalVM JDK](https://graalvm.org/downloads/) or newer
+
+ [^1]: JDK 17 is supported with interpreter only, JDK 21 and newer with JIT compilation.
```
Comment (steve-s) [inline graalpy/graalpy-javase-guide/README.md:23]: I still think we should warn people against trying with GraalVM for JDK 17. Maybe something like:
```suggestion
 [^1]: Oracle/Open/GraalVM JDK 21 or later are supported with JIT compilation, Oracle/Open JDK 17 are supported with interpreter only. GraalVM for JDK 17 is not supported.
```

**Review by steve-s — APPROVED** [review]

**Comment by steve-s** [comment]
@msimacek @fniephaus all todos should be resolved. We will update the "Next steps" sections with more links to in depth docs later when they are done. For now, this should be good to go. I've also added 3 new guides:

- native extensions guide (already reviewed in bb)
- freezing Python dependencies guide (already reviewed in bb)
- custom venv guide: (new) people repeatedly ask about this, and we also often tell people to try manually created venv, when installation of some package via Maven fails, so I think it would be handy to have a document that we can point people to

File: graalpy/graalpy-javase-guide/README.md (hunk 1 above)
Comment (steve-s) [reply graalpy/graalpy-javase-guide/README.md]: I took the liberty and applied similar wording

**Comment by fniephaus** [comment]
Thanks, @steve-s! I have moved all guides over to https://github.com/graalvm/graal-languages-demos/tree/main/graalpy.

**Pull request closed without merging**